presto-product-tests/bin/run_on_docker.sh

```diff
@@ -35,6 +39,18 @@ function stop_unnecessary_hadoop_services() {
   docker exec ${HADOOP_MASTER_CONTAINER} supervisorctl stop zookeeper
 }
 
+function start_mapr_httpfs_service() {
+  HADOOP_MASTER_CONTAINER=$(hadoop_master_container)
+  docker exec ${HADOOP_MASTER_CONTAINER} chown -R mapr ${MAPR_HOME}/httpfs
+  docker exec ${HADOOP_MASTER_CONTAINER} maprcli node services -name httpfs -action start -nodes ${HADOOP_HOST_NAME}
+}
+
+function configure_maprclient() {
+  HADOOP_MASTER_CONTAINER=$(hadoop_master_container)
```

Comment:
after getting more context from our slack conversation, I see that the `/server/configure.sh` should be run from a specialized `mapr-presto-launcher-wrapper.sh`. The variables used in this function should be defined in that script. Please don't bother with parametrizing the 'HADOOP_HOST_NAME', it's hardwired everywhere anyway. It's ok to just define the variable in that launcher script.

The specialized wrapper should be used by having a `docker/common/mapr.yml` compose file, added to the `mapr` profile. The `image:` override should take place there as well.